js/search.js

```diff
@@ -968,6 +968,7 @@
 						} else */ if (learnsetEntry.indexOf(gen) < 0) {
 							continue;
 						}
+						if (this.gen === 8 && BattleMovedex[moveid].isNonstandard === "Past") continue;
```

Comment:
Well, we'll need to add an exception for NatDex formats.

Can we do that now, actually? Will make it easier later.

Reply:
I mean, it is possible, but I believe @TheImmortal said somewhere that illegal moves wouldn't be usable in NatDex metas, just the Pokemon. Correct me if im wrong.

Reply:
I was planning on including illegal moves, though...

Reply:
I was going to just blanket-allow `Past` in the NatDex meta.

Reply:
do you know what the formats will be named? maybe if its like [Gen 8 NatDex] i can add a format based exception

Reply:
`[Gen 8] National Dex` is the tentative name

Reply:
did